src/edgeworkers/ew-service.ts

```diff
@@ -103,6 +103,14 @@ export function getContracts() {
   return httpEdge.getJson(`${EDGEWORKERS_API_BASE}/contracts`).then(r => r.body).catch(err => error.handleError(err,"GET_CONTRACT"));
 }
 
+export function getProperties(ewId: string, activeOnly: boolean) {
+  let qs: string = "?activeOnly=true";
+  if (activeOnly === undefined) {
+    qs = "";
+  }
```

Comment:
can we reverse this ? For instance lets define it with empty string and if its not undefined then add this `"?activeOnly=true";`